hail/src/test/scala/is/hail/io/compress/BGzipCodecSuite.scala

```diff
@@ -114,4 +115,57 @@ class BGzipCodecSuite extends SparkSuite {
     }
     p.check()
   }
+
+  @Test def testVirtualSeek() {
+    sc.hadoopConfiguration.setLong("mapreduce.input.fileinputformat.split.minsize", 1L)
+    // real offsets of the start of each block
+    val blockStarts = Array[Long](14653, 28034, 40231, 55611, 69140, 82949, 96438, 110192, 121461, 133703,
+      146664, 158711, 171239, 181362)
+    // offsets into the uncompressed file
+    val uncompBlockStarts = Array[Long](65280, 130560, 195840, 261120, 326400, 391680, 456960, 522240,
+      587520, 652800, 718080, 783360, 848640, 913920)
+
+    val blockPointerOffset = 16 // magic number for creating virtual file pointers into bgzipped files
```

Comment:
Use your'e going to use BlockCompressedFilePointerUtil, use makeFilePointer instead?

Reply:
In particular, it does checking in case the block offset >= 64K.

Reply:
I can't seem to get that to work in the tests. When I try, I get the following error:

```
/home/cdv/work/src/hail/hail/src/test/scala/is/hail/io/compress/BGzipCodecSuite.scala:147: method makeFilePointer in object BlockCompressedFilePointerUtil cannot be accessed in object htsjdk.samtools.util.BlockCompressedFilePointerUtil
      val vptr = BlockCompressedFilePointerUtil.makeFilePointer(cOff, extra)
```

Reply:
Ugh, it's private.  These functions are just a line or two, I'd probably just make our own version.

I also don't like the term pointer for a file.  I'd call it an offset or a position.

Reply:
It's not private though. I can call the method from java just fine. I can't call it from scala, and I don't know why.

Reply:
It is private.  Java is private by default, and it has no public modify in the version I'm looking at.

No access modifier in Java means accessible from the same package.  Are you calling it from Java in `htsjdk.samtools.util`?

Reply:
Oh, the method went from no modifier to public between 2.14.2 which we are using and the current version of the source. We could try upgrading the dependency?